### PR TITLE
Move Harbor Upgrading Notes to chart-docs

### PR DESCRIPTION
**Pull request by juan131**
This PR moves the Harbor "Upgrading Notes" to https://docs.bitnami.com

Follow-up https://github.com/bitnami/charts/pull/9263, reducing the README.md size.

### CODE REVIEW

File: charts/harbor/_upgrade.md.erb [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+
+### To 12.0.0
+
+This major release renames several values in this chart and adds missing features, in order to be inline with the rest of assets in the Bitnami charts repository. Additionally updates the PostgreSQL & Redis subcharts to their newest major 11.x.x and 16.x.x, respectively, which contain similar changes.
+
+#### What changes were introduced in this major version?
+
+* *harborAdminPassword* was renamed to *adminPassword* and *forcePassword* is deprecated.
+* Traffic exposure was completely redesign:
+  * There's a new *exposureType* parameter has been added that allows deciding whether to expose Harbor using Ingress or a NGINX proxy.
```
Comment (migruiz4) [inline charts/harbor/_upgrade.md.erb:20]: This sentence is missing a 'that' before 'has been added', but maybe it is too wordy. I suggest:
```suggestion
  * The new parameter *exposureType* allows deciding whether to expose Harbor using Ingress or an NGINX proxy.
```

**Review by migruiz4 — APPROVED** [review]
Looks good to me, but please have a look at the typos and suggestions below.

**Comment by juan131** [comment]
Thanks @migruiz4 !
